### PR TITLE
Update agenda for meetup 8 and 9

### PR DESCRIPTION
**Pull request by aakankshaduggal**
This PR updates agenda for meetup 8 (#51) and meetup 9(#46). It also updates the agenda for the previous meetups' slides

### CODE REVIEW

File: docs/content.md [hunk 1]
```diff
@@ -2,54 +2,64 @@
...
 ## Upcoming 
 
+April 5th, 2022
+
+**AI4CI - Artificial Intelligence for Continuous Integration**
+
+* By Aakanksha Duggal, Hema Veeradhi, Karan Chauhan, Michael Clifford, Oindrilla Chatterjee and Shrey Anand
+* In this session, the speakers introduce AI4CI (Artificial Intelligence for Continuous Integration), a collection of open-source AIOps tools involving open operations data made available by Kubernetes testing and visualization platforms. 
+* The speakers will demonstrate how they developed a set of Jupyter Notebooks which are automated using Kubeflow Pipelines into repeatable processes that collect data from various open CI/CD data platforms, calculate and visualize key performance indicator metrics, build AI/ML services to support operations and create a reproducible workflow around these processes.
+
```
Comment (oindrillac) [inline docs/content.md:12]: Can we add something like
```suggestion
In this session, attendees learn about a set of AIOps services deployed on Operate First that they can start using and get familiarized with how to use the set of tools available on the Operate First cloud to deploy their own workloads . 
```

**Review by oindrillac — CHANGES_REQUESTED** [review]

File: docs/content.md [hunk 2]
```diff
@@ -2,54 +2,65 @@
...
+* By Aakanksha Duggal, Hema Veeradhi, Karan Chauhan, Michael Clifford, Oindrilla Chatterjee and Shrey Anand
+* In this session, the speakers introduce AI4CI (Artificial Intelligence for Continuous Integration), a collection of open-source AIOps tools involving open operations data made available by Kubernetes testing and visualization platforms. 
+* The speakers will demonstrate how they developed a set of Jupyter Notebooks which are automated using Kubeflow Pipelines into repeatable processes that collect data from various open CI/CD data platforms, calculate and visualize key performance indicator metrics, build AI/ML services to support operations and create a reproducible workflow around these processes.
+* In this session, attendees learn about a set of AIOps services deployed on Operate First that they can start using and get familiarized with how to use the set of tools available on the Operate First cloud to deploy their own workloads.
+
+## Past Events
+
 March 22nd, 2022
 
 **Sandiego**
```
Comment (oindrillac) [inline docs/content.md:18]: nit - formatting for this session is different from others. Date and title are on different lines/

Comment (oindrillac) [inline docs/content.md:18]: nit: formatting for title and date for this session is different from others. should be on same line?

**Review by oindrillac — CHANGES_REQUESTED** [review]
small nit, otherwise lgtm

**Comment by aakankshaduggal** [comment]
Thanks for the review @oindrillac , fixed it :+1:

**Review by oindrillac — APPROVED** [review]
/lgtm
/approve

**Pull request merged**